Added method to check whether Start has been called

clr182 opened the pull request:
## Goal

<!-- Why is this change necessary? -->
To keep inline with other notifiers

## Changeset

<!-- What changed? -->
Added method to BugsnagFunctionLibrary.cpp and static bool that is set to true when the Start() method is called.

## Testing

<!-- How was it tested? What manual and automated tests were
     run/added? -->
Tested manually using Debug logs

clr182 committed:
Added method to check whether Start has been called

clr182 committed:
update clang formating

clr182 committed:
update clang formating part 2

richardelms commented on Plugins/Bugsnag/Source/Bugsnag/Private/BugsnagFunctionLibrary.cpp:
```diff
@@ -79,6 +81,7 @@ void UBugsnagFunctionLibrary::Start(const TSharedRef<FBugsnagConfiguration>& Con
 
 	GPlatformBugsnag.Start(Configuration);
 
+	Started = true;
```
I would move this to after Delegates are setup

richardelms commented on Plugins/Bugsnag/Source/Bugsnag/Private/BugsnagFunctionLibrary.cpp:
```diff
@@ -353,6 +356,20 @@ TArray<TSharedRef<const IBugsnagBreadcrumb>> UBugsnagFunctionLibrary::GetBreadcr
 #endif
 }
 
+bool UBugsnagFunctionLibrary::IsStarted()
```
is this logging necessary?
maybe just return Started;

clr182 committed:
Removed debug logging

Removed debug logging and moved started trigger to after delegates in configuration

clr182 committed:
fixed linting

clr182 committed:
fix typo

clr182 committed:
Merge branch 'next' into PLAT-8569-IsStartedMethod
